model/Web/ImagesProcessed.php

```diff
@@ -7,7 +7,7 @@
 use Gamecon\XTemplate\XTemplate;
 use Nahled;
 
-readonly class Loga
+readonly class ImagesProcessed
```

Comment:
Já vím že ta čengliš v kódu je náročná, kor když je programátor od prvních dní cepovaný na angličitnu :D

Ale poprosím o větší konzistenci v "buďto celé anglicky", nebo "celé cesky".

Teď je to kostrbaté `ImagesProcessed::logaSponzoruPrehled()`